flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -302,9 +320,10 @@ private void commitPendingResult(
 
   private void deleteCommittedManifests(
       List<ManifestFile> manifests, String newFlinkJobId, long checkpointId) {
+    FileIO fileIO = table.io();
```

Comment:
nit: any reason this `fileIO` variable is needed here?

Reply:
Initially the was to ensure the same file IO was used in the loop. I removed this, now that the table load doesn't happen in the supplier's `get()` method anymore.